cloud

```diff
@@ -1243,6 +1243,22 @@ function pubsub_withdraw() {
     pubsub_topic_withdraw "$project" "$pick_notifications_trigger_topic"
 }
 
+# Deploy a Google Cloud Storage Bucket
+# Args: project storage_bucket
+function storage_deploy() {
+    declare -r project="$1"; shift
+    declare -r storage_bucket="$1"; shift
```

Comment:
Once we passed the storage bucket to this (and the other function), it's clear that it's a **storage** bucket, so we don't need to have `storage_` in the variable name. Please name it just `bucket` in these functions (but not in the calling function, where there is more context we need to separate it from).